flake/dev/devshell.nix

```diff
@@ -108,7 +108,7 @@
 
                 echo -e "\n=> You can then open your browser to view the doc\n"
 
-                (cd "$doc_derivation"/share/doc && ${pkgs.lib.getExe pkgs.python3} ${./server.py})
+                (cd "$doc_derivation" && ${pkgs.lib.getExe pkgs.python3} ${./server.py})
```

Comment:
To avoid needing to backport https://github.com/nix-community/nixvim/pull/3349, I've updated this line instead.